woocommerce/woocommerce-gutenberg-products-block
# Add Empty Cart View with inner blocks

Fixes #1296.

### Screenshots

![imatge](https://user-images.githubusercontent.com/3616980/70431876-468bf400-1a7e-11ea-9a4a-db4ddcbfe891.png)

### How to test the changes in this Pull Request:

First, modify [this line](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/1350/files#diff-778249010b69d6b2cd0540c40641b375R12) so the cart appears always empty in the frontend:

```diff
-const isCartEmpty = false;
+const isCartEmpty = true;
```

1. Create a post with the _Cart_ block.
2. Switch to the _Empty Cart_ tab and play around with the inner blocks (modify them, remove some, add new ones, etc.).
3. Preview the post in the frontend and verify everything looks good.
